src/hvps/commands/iseg/__init__.py

```diff
@@ -50,33 +52,37 @@ def _parse_response(response: bytes, expected_response_type: type | None) -> Lis
     """
 
     try:
-        response: str = response.decode("ascii").strip()
+        response = response.decode("ascii").strip()
     except UnicodeDecodeError:
         raise ValueError(f"Invalid response: {response}")
 
     if expected_response_type == float or expected_response_type == List[float]:
         # pattern for a float in scientific notation followed or not by units
         pattern = (
-            r"^[-+]?[0-9]*\.?[0-9]+([eE][-+]?[0-9]+)?(\s*[a-zA-Z]+(/+[a-zA-Z]+)?)?$"
+            r"^[-+]?[0-9]*\.?[0-9]+([eE][-+]?[0-9]+)?(\s*[a-zA-Z%]+(/+[a-zA-Z]+)?)?$"
```

Comment:
why this change to the regex? is this intentional?

Reply:
A monitor command concerning ramp speed returns a percentage, the regex needs to expect that